Test the role via molecule and update supported OS

tobiashuste opened the pull request:
* Manage dependencies via pipenv
* Run Molecule via GitHub Actions
* Update the list of support OS

Closes #8

tobiashuste commented:
@Normo I would be happy if you could have a look. This implements basic role testing via molecule. There is no verification step implemented yet. It simply takes the default installation and deploys it in one of the container.

Normo reviewed (changes requested):
Thank you very much @tobiashuste for implementing this! :star_struck: Very much appreciate the effort. I've added some comments/suggestions.

Normo commented on .github/workflows/ci.yml:
```diff
@@ -0,0 +1,74 @@
+  pull_request:
+  push:
+    branches:
+      - "main"
```
What do you think @tobiashuste ? Shall we rename the default branch to `main`?

tobiashuste replied:
Probably we should do this.

Normo replied:
:ok_hand: let's do it with #13

Normo commented:
@tobiashuste We can take inspiration for the verfication step from the test play in [tests/test.yml](https://github.com/hifis-net/ansible-role-unattended-upgrades/blob/master/tests/test.yml#L28)

tobiashuste commented:
> @tobiashuste We can take inspiration for the verfication step from the test play in [tests/test.yml](https://github.com/hifis-net/ansible-role-unattended-upgrades/blob/master/tests/test.yml#L28)

@Normo I implemented the verification step according to the existing tests. I also changed your other suggestions.

So far I did not touch the branch name to wait for your feedback.

Normo reviewed (approved):
Thanks, @tobiashuste LGTM :+1:
